cypress/support/step_definition/tasks/createEditAndDeleteTasks.spec.js

```diff
@@ -2,21 +2,13 @@ import Tasks from '../../pages/tasks/pageTasks'
 
```

Comment:
file name can be --> createEditDeleteSubGradFinishRestoreTaskSteps.spec.js

Reply:
done

Reply:
done